README.md

```diff
@@ -1,79 +1,95 @@
-These include `AWS_ACCESS_KEY_ID`, `AWS_SECRET_ACCESS_KEY`, `S3_BUCKET`, `S3_ENDPOINT_URL`, `CEPH_BUCKET_PREFIX`, `REPO` and `ORG`.
+This will initiate the model training and inference action. 
 
-Now, go to the "Actions" section of your forked repository and click on workflow "Run in container" and click on "Run workflow"
 
-<img width="500" alt="image" src="https://user-images.githubusercontent.com/32435206/195928717-079b0c85-c953-43a9-b6e4-cf6efbc7dff5.png">
+2. **Inference Mode** : 
 
-## Step 4
+Similar to the `train-ttm.yaml` file, you can add another file called `predict-ttm.yaml` file to `.github/worklows/` like [this](https://github.com/aicoe-aiops/ocp-ci-analysis/blob/master/.github/workflows/predict-ttm.yaml). This file has `MODE` as `0` which would enable just inference on the new incoming Pull Request and add a comment to the pull request specifying the approximate time it will take to be merged.
 
-To view your running workflow from the Github UI, go to "Actions" and click on the workflow run
+![image](https://user-images.githubusercontent.com/26301643/206541965-c85eb5f8-012e-454c-9f0d-467db0c8be07.png)
 
-<img width="700" alt="image" src="https://user-images.githubusercontent.com/32435206/196230588-d8485c8c-2f97-46a9-ba07-b86e1b7c6782.png">
+
+## Step 3
+
+To view your running workflow from the Github UI, go to "Actions" and click on the workflow run :
+
+![image](https://user-images.githubusercontent.com/26301643/206549380-369e6680-5e4e-4994-ae8a-59fd149db426.png)
+
+Click on `pipeline` to see logs and errors :
+
+![image](https://user-images.githubusercontent.com/26301643/206549621-0184880a-769e-48dc-928f-44c721464999.png)
 
 ## Architecture
 
-![ttm github workflow (1)](https://user-images.githubusercontent.com/32435206/196721278-b668b411-d1b6-4af3-8d73-3cb0dd8d58ab.png)
+![ttm github workflow (1)](https://user-images.githubusercontent.com/26301643/206780355-7d4e877c-c9ed-448d-8331-ad31e865b85f.png)
+
+## Alternate Approach
+
+You can also use this tool for your repository using an alternate approach. Here are the steps to follow :
+
+1. Fork this [repository](https://github.com/redhat-et/time-to-merge-tool) and to your fork add the secrets as mentioned [here](https://github.com/redhat-et/time-to-merge-tool#step-1). Make sure to mention the `REPO` and `ORG` for the repository you want to run TTM on.
+2. Go to `Actions` for your fork and select the workflow you want to run. You can also mention the mode of action by changing it [here](https://github.com/redhat-et/time-to-merge-tool/blob/016897ab03a9ff0aa800d6bcaececad028c93841/.github/workflows/inference.yaml#L22)
+<img width="1056" alt="Screen Shot 2022-12-09 at 12 43 26 PM" src="https://user-images.githubusercontent.com/26301643/206761255-d74f83d9-183e-4ad9-97da-936f61db36a2.png">
+3. You can also interact with this tool by POST request to Github API endpoint. From your terminal, clone your repository and run `bash run-ttm.sh` .
```

Comment:
```suggestion
3. You can also interact with this tool by POST request to Github API endpoint. From your terminal, clone your repository and run `bash run-ttm.sh` . This will run the training workflow and train the TTM model on the repo and org of your choice.
```